docs/boundary-services-a-to-z.adoc

```diff
@@ -1,49 +1,12 @@
 = Boundary Services A-Z
 
-This document describes how to run an environment with boundary services.
-It's a quick rundown and assumes knowledge of the basic setup described in the
-Running Omicron (Non-Simulated) document.
+NOTE: The instructions for _deploying_ SoftNPU with Omicron have been folded into xref:how-to-run.adoc[the main how-to-run docs].
```

Comment:
@rcgoodfellow I mentioned this in the PR description but I think most (but not all) of this doc is now covered by how-to-run so I trimmed this a lot but left the stuff that went a little deeper than what's in how-to-run.  What do you think?

Reply:
Yeah, this looks good to me. No need to keep the redundant prose around.